### ISSUE TITLE
[UI/UX] problem with disassembly page.

### ISSUE THREAD

**Issue by WardenGnaw** [microsoft/vscode#131655]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.59.1
- OS Version:  Windows 7 x64 SP1

Steps to Reproduce:

# In disassembly page, the width of the yellow background is less than the width of the blue border line.
**Was it originally designed to be like this? Or is this a bug?**

![](https://i.bmp.ovh/imgs/2021/08/73acb9090e87b419.png)

**My editor font size setting is 20**

Originally from @thefatwhale at https://github.com/microsoft/vscode-cpptools/issues/8051

**Comment by connor4312** [comment]
I don't think this should happen. The leftmost column if effectively the "gutter" and is where breakpoint lines and indicators are shown. Having the background apply to that region would introduce contrast issues (both the line background and stack frame indicator are, by default, yellow)

**Comment by roblourens** [comment]
I think @WardenGnaw meant "height", I assume this is about the gap at the bottom of the row

**Comment by connor4312** [comment]
Ah, gotcha. Yea that should be fixed.
